plugins/generator-1.21.4/neoforge-1.21.4/item.definition.yaml

```
@@ -4,15 +4,18 @@ templates:
   - template: item/item_container.java.ftl
     name: "@SRCROOT/@BASEPACKAGEPATH/item/inventory/@NAMEInventoryCapability.java"
     condition: hasInventory()
+  - template: item/item_render.java.ftl
+    name: "@SRCROOT/@BASEPACKAGEPATH/client/renderer/@NAMEItemRenderer.java"
+    condition: hasJavaModel()

```

The definition part related to the attached `item_renderer.java.ftl` file (`item` sub-package is already used to store our custom `ItemModel` subclass):
```suggestion
  - template: item/item_renderer.java.ftl
    name: "@SRCROOT/@BASEPACKAGEPATH/client/renderer/item/@NAMEItemRenderer.java"
    condition_any:
      - hasJavaModel()
      - "${data.getModels()?filter(e -> e.hasJavaModel())?has_content}"
```
The other occurrence of that template's reference in this file should then be removed.

---

Ok

---

Done